benchmarks/definition/config.py

```diff
@@ -88,3 +89,15 @@ def __call__(self) -> Result:
         )
 
         return Result(self.name, result, metadata)
+
+
+# Register un-/structure hooks
+converter.register_unstructure_hook(
```

Comment:
Confused here, this looks to me like there is no special handling for the actual `callable`, I thought this should be removed here? Or am I misunderstanding what this does?